docs/GovernanceModel.md

```diff
@@ -37,14 +37,20 @@ Remember that security issues should be reported through a separate channel, and
 will receive a response within 24 hours. See [Reporting Security
 Issues](Contributing.md#reporting-security-issues).
 
-Maintainers and Contributors
-----------------------------
-
-We currently define "maintainer" as anyone with direct write access to the Open
-Enclave repository on GitHub, and "contributor" as anyone else. For now, the
-only maintainers are Microsoft employees, but we hope to expand this in the
-future to include consistent contributors. Our process for doing so is not yet
-defined, but if you are interested, let us know!
+Maintainers, Committers, and Contributors
+-----------------------------------------
+
+We define "maintainer" as members of the Open Enclave "Maintainers Committee",
+as listed in the [maintainers document](Maintainers.md). A "committer" is anyone
+with direct write access to the Open Enclave repository on GitHub, as granted by
+the maintainers committee. All maintainers are committers, but not all
+committers are maintainers. Finally, "contributor" is anyone else making
+contributions to the project, including: creating or commenting on issues,
+opening or reviewing pull requests, or other useful contributions such as
+providing support in forums or chats.
+
+See the [maintainers document](Maintainers.md) for the current Maintainers
```

Comment:
Copyedit: Consider inlining this with the definition of the maintainer to preserve flow:

> We define a _maintainer_ as any member of the Open Enclave Maintainers Committee. The current members of this committee are listed in the [maintainers document](Maintainers.md), which also describes our process for adding new maintainers and committers. A _commiter_ is anyone granted direct write access to the Open Enclave repository on GitHub by the maintainers, so all maintainers are committers, but not all committers are maintainers. Finally, a _contributor_ is anyone engaged in improving the project, for example, by creating or commenting on issues, opening or reviewing pull requests, providing support in forums or chats, and so on.